Sources/Lumina/UI/LuminaCameraViewSettings.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  LuminaCameraViewSettings.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+/// The position of the camera that is active on Lumina
+public enum CameraPosition: String {
+  /// the front facing camera of the iOS device
+  case front
+  /// the back (and usually main) facing camera of the iOS device
+  case back
+}
+
+/// The resolution to set the camera to at any time - refer to AVCaptureSession.Preset definitions for matching, closest as of iOS 11
+public enum CameraResolution: String, CaseIterable {
+  case low352x288 = "Low 352x288"
+  case vga640x480 = "VGA 640x480"
+  case medium1280x720 = "Medium 1280x720"
+  case high1920x1080 = "HD 1920x1080"
+  case ultra3840x2160 = "4K 3840x2160"
+  case iframe1280x720 = "iFrame 1280x720"
+  case iframe960x540 = "iFrame 960x540"
+  case photo = "Photo"
+  case lowest = "Lowest"
+  case medium = "Medium"
+  case highest = "Highest"
+  case inputPriority = "Input Priority"
+
+  // swiftlint:disable cyclomatic_complexity
+  func foundationPreset() -> AVCaptureSession.Preset {
+    switch self {
+      case .vga640x480:
+        return AVCaptureSession.Preset.vga640x480
+      case .low352x288:
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/f3c4aa8d5c66dd9d2660cdba8e658b8902c9752f/Sources/Lumina/UI/LuminaCameraViewSettings.swift#L40"></span>Case statements should vertically align with their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>LuminaCameraViewSettings.swift:40</code></td>
    </tr>
  </tbody>
</table>